include/thread_pool/thread_pool.hpp

```diff
@@ -13,10 +15,13 @@
 namespace tp
 {
 
+template <typename Task>
+using ConcurrentQueue = moodycamel::ConcurrentQueue<Task, moodycamel::ConcurrentQueueDefaultTraits>;
```

Comment:
Can we do a quick performance benchmark comparing `MPMCBoundedQueue` and `ConcurrentQueue`?
Just to be sure that moodycamel is faster...